src/Microsoft.AspNetCore.Mvc.RazorPages/ApplicationModels/PageRouteModel.cs

```diff
@@ -73,5 +71,18 @@ public PageRouteModel(PageRouteModel other)
         /// Gets the <see cref="SelectorModel"/> instances.
         /// </summary>
         public IList<SelectorModel> Selectors { get; }
+
+        /// <summary>
+        /// Gets a collection of route values that must be present in the <see cref="RouteData.Values"/> 
+        /// for the corresponding page to be selected.
+        /// </summary>
+        /// <remarks>
+        /// <para>
+        /// The value of <see cref="ViewEnginePath"/> is considered an implicit route value corresponding
+        /// to the key <c>page</c>. These entries will be added to <see cref="ActionDescriptor.RouteValues"/>, 
```

Comment:
`These entries will be added to` -> `These entries will be implicitly added to ... when the action descriptor is created...`